src/libstd/time.rs

```diff
@@ -114,6 +129,21 @@ pub struct Instant(time::Instant);
 ///    }
 /// }
 /// ```
+///
+/// # Underlying System calls
+/// Currently, the following system calls are being used to get the current time using `now()`:
+///
+/// |  Platform |               System call                        |
+/// |:---------:|:------------------------------------------------:|
+/// | Cloud ABI | [clock_time_get (Realtime Clock)](https://github.com/NuxiNL/cloudabi/blob/master/cloudabi.txt) |
+/// |    SGX    | Not implemented                                  |
+/// |    UNIX   | [gettimeofday](http://man7.org/linux/man-pages/man2/gettimeofday.2.html)                           |
```

Comment:
As in the other table, this is only true for Mac and iOS. Other unix targets use `clock_gettime` with `CLOCK_REALTIME`.